ethstorage/es-node
# Change the order in which peers are assigned to tasks

Fix issue (https://github.com/ethstorage/es-node/issues/90), which causes by the way assign peers to tasks (see the issue for more detail). Use the solution 2 in the issue 90.

How to test and verify:
Run the es-node using `run.sh`. and see the sync log like the following:
`INFO [11-29|22:16:09.439] Storage sync in progress                 progress=54.29% peerCount=0 syncTasksRemain=1@32 blobsSynced=4,165,248@508.45GiB blobsToSync=3,834,773 fillTasksRemain=0 emptyFilled=388,587@47.43GiB emptyToFill=0 timeUsed=6h43m57.096s eta=9h52m47.745s`
After most of the blobs synced (like blobsSynced=4,165,248), the remaining task remained the same as init (like syncTasksRemain=1@32). That means p2p concurrency would not reduce during syncing.
